### PR TITLE
improvements about spring remote debugging

### PR DESCRIPTION
**Pull request by ivywei0125**
What does this implement/fix? Explain your changes.
---------------------------------------------------
- update icons
- rename file to make action clearer
- show "learn more" link when enable debugging
- save before run task data persistent

Does this close any currently open issues?
------------------------------------------

Any relevant logs, screenshots, error output, etc.?
-------------------------------------

Any other comments?
-------------------
…

Has this been tested?
---------------------------
- [ ] Tested

### CODE REVIEW

File: PluginsAndFeatures/azure-toolkit-for-intellij/azure-intellij-plugin-springcloud/src/main/java/com/microsoft/azure/toolkit/intellij/springcloud/remotedebug/PortForwardingTaskProvider.java [hunk 1]
```diff
@@ -87,20 +92,51 @@ protected PortForwarderBeforeRunTask(RunConfiguration config) {
...
+            return Optional.ofNullable(account.getTokenCredential(this.state.properties.get(PortForwarderBeforeRunTaskState.SUBSCRIPTION_ID))
+                    .getToken(request).block())
+                    .map(AccessToken::getToken).orElse("");
         }
+    }
+
+    @Getter
+    @Setter
+    static class PortForwarderBeforeRunTaskState {
+        private Map<String, String> properties = new HashMap<>();
```
Comment (ivywei0125) [inline PluginsAndFeatures/azure-toolkit-for-intellij/azure-intellij-plugin-springcloud/src/main/java/com/microsoft/azure/toolkit/intellij/springcloud/remotedebug/PortForwardingTaskProvider.java:117]: resource id

File: Utils/azure-toolkit-ide-libs/azure-toolkit-ide-springcloud-lib/src/main/java/com/microsoft/azure/toolkit/ide/springcloud/SpringCloudActionsContributor.java [hunk 2]
```diff
@@ -134,10 +134,11 @@ public void registerGroups(AzureActionManager am) {
             ResourceCommonActionsContributor.RESTART,
             ResourceCommonActionsContributor.DELETE,
             "---",
-            SpringCloudActionsContributor.STREAM_LOG_APP,
             SpringCloudActionsContributor.ENABLE_REMOTE_DEBUGGING,
             SpringCloudActionsContributor.DISABLE_REMOTE_DEBUGGING,
-            SpringCloudActionsContributor.ATTACH_DEBUGGER_APP
+            SpringCloudActionsContributor.ATTACH_DEBUGGER_APP,
```
Comment (ivywei0125) [inline Utils/azure-toolkit-ide-libs/azure-toolkit-ide-springcloud-lib/src/main/java/com/microsoft/azure/toolkit/ide/springcloud/SpringCloudActionsContributor.java]: change order

**Review by wangmingliang-ms — DISMISSED** [review]
let's merge this pr first,  remember to fix comments in later PRs.

File: PluginsAndFeatures/azure-toolkit-for-intellij/azure-intellij-plugin-springcloud/src/main/java/com/microsoft/azure/toolkit/intellij/springcloud/remotedebug/PortForwardingTaskProvider.java [hunk 3]
```diff
@@ -109,16 +106,15 @@ public PortForwarderBeforeRunTaskState getState() {
         @Override
         public void loadState(@Nonnull PortForwarderBeforeRunTaskState state) {
             XmlSerializerUtil.copyBean(state, this.state);
+            final String resourceId = this.state.properties.get(PortForwarderBeforeRunTaskState.RESOURCE_ID);
+            this.appInstance = (SpringCloudAppInstance) Azure.az().getById(resourceId);
```
Comment (ivywei0125) [inline PluginsAndFeatures/azure-toolkit-for-intellij/azure-intellij-plugin-springcloud/src/main/java/com/microsoft/azure/toolkit/intellij/springcloud/remotedebug/PortForwardingTaskProvider.java:110]: log out and test
get resource when run config

**Review by Flanker32 — APPROVED** [review]
LGTM

**Pull request merged**